### ISSUE TITLE
create jvm-profiling-tools/jbang-catalog

### ISSUE THREAD

**Issue by maxandersen** [jvm-profiling-tools/ap-loader#19]
At the moment we have jbang aliases for `jvm-profling-tools/ap-loader` with a jbang-catalog.

Means that you need to do `jbang <alias>@jvm-profiling-tools/ap-loader` for everything.

If there was a jvm-profling-tools/jbang-catalog repo we could shorten this to just: `jbang <alias>@jvm-profiling-tools`

shorter and easier to remember imo.

wdyt?

**Comment by parttimenerd** [comment]
@jvm-profiling-tools/administrators what do you think? I don't have a problem with it.

**Comment by apangin** [comment]
IIUC, you propose creating a separate repository under `jvm-profiling-tools` org to hold jbang aliases for jattach, async-profiler and related tools.

I'm always happy to see more ways to facilitate adoption of async-profiler and friends, however, I find it a bit confusing to do it this way. If we are talking about accessing profiler commands via jbang, isn't it counterintuitive to connect them through an external location, which is neither a home for jbang nor for the profiler? Currently, jbang-catalog is at least tied with ap-loader that provides those commands. But when it becomes a separate repository, it will no longer be tightly connected with the project, making maintenance unobvious and complicated.

**Comment by parttimenerd** [comment]
Good point. I'll close this issue.

**Comment by maxandersen** [comment]
Just note - the maintanence is close to zero as the alias point to the latest. 

If there ever was a need to maintain more often it could be easily automated.
